src/appleseed/renderer/meta/tests/test_sss.cpp

```diff
@@ -57,26 +57,36 @@ using namespace std;
 
 TEST_SUITE(Renderer_Modeling_BSSRDF_SSS)
 {
+    template <typename ComputeRdFun>
     double rd_alpha_prime_roundtrip(
         const double rd,
         const double eta)
     {
-        const double two_c1 = fresnel_moment_two_c1(eta);
-        const double three_c2 = fresnel_moment_three_c2(eta);
-
-        const double alpha_prime = compute_alpha_prime(rd, two_c1, three_c2);
-        return compute_rd(alpha_prime, two_c1, three_c2);
+        ComputeRdFun f(eta);
+        const double alpha_prime = compute_alpha_prime(f, rd);
+        return f(alpha_prime);
     }
 
     TEST_CASE(Rd_AlphaPrime_Roundtrip)
     {
-        EXPECT_FEQ_EPS(0.0, rd_alpha_prime_roundtrip(0.0, 1.6), 0.001);
-        EXPECT_FEQ_EPS(0.1, rd_alpha_prime_roundtrip(0.1, 1.3), 0.001);
-        EXPECT_FEQ_EPS(0.2, rd_alpha_prime_roundtrip(0.2, 1.2), 0.001);
-        EXPECT_FEQ_EPS(0.4, rd_alpha_prime_roundtrip(0.4, 1.3), 0.001);
-        EXPECT_FEQ_EPS(0.6, rd_alpha_prime_roundtrip(0.6, 1.4), 0.001);
-        EXPECT_FEQ_EPS(0.8, rd_alpha_prime_roundtrip(0.8, 1.3), 0.001);
-        EXPECT_FEQ_EPS(1.0, rd_alpha_prime_roundtrip(1.0, 1.5), 0.001);
+        EXPECT_FEQ_EPS(0.0, rd_alpha_prime_roundtrip<ComputeRd>(0.0, 1.6), 0.001);
```

Comment:
These are pretty low epsilons, especially given that everything is double precision. Can't get them any lower? Are you sure the roundtrip works as expected?